espresso-tests/.env

```diff
@@ -0,0 +1,21 @@
+# env vars for chain name and rpc_url
+CHAIN_NAME="1337"
```

Comment:
`CHAIND_ID`, also we should prefix everything from L1 with `L1`.

Reply:
I think renaming everything to parent and child chain makes sense as L1 and L2 don't necessarily make sense in the context of orbit chains.